### PR TITLE
Added documentation to support servicenow click to dial

### PR DESCRIPTION
**Pull request by AmardeepKumar1205**
## Description

- Added documentation to setup service now click to dial

## Checklist

Please ensure that you have completed the following tasks before submitting this pull request:

- [x] I have updated the documentation or README if necessary.
- [ ] My code follows the project's coding style guidelines.
- [ ] My changes are self reviewed

## Screenshots (if applicable)

If your changes include UI/UX modifications or additions, please include relevant screenshots here.

## Additional Notes

None

### CODE REVIEW

File: documentation/ServiceNowSetup.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
...RoPgLk71YzMu7P1vS6NnLnAotuWAnfc1zlwiyrjvPo7X/6Ek/t8+yrzXId/42175WVm+ph84V3XakUBOg665tG1Tx+T9diKynHmByLMBCGX6G+FhA6rV6+M+g3qoGatypwW0l+kg8003mkhcPhqZ6upF/r3yMvM1Eu6vPw2jTVe3KBrHV3z6NpHWNvHezhXyBgRZoKQS/TF2HohIrvrrpsRqsJatau46hR3dNuomJGRz2S0Zfe+CXmJq519tDWfH7M8czPr85IFi96n4ggdN13jKlcqz9c89/nxNCHryOR/QcgB9LHRHx273e5xE4A2ugmAwm+/nYUd23fjfFIyTp+KU3nGz8IIgpAe/mRk+Pmgjt6MCgUCCa2wsBCeU0bDl/Ub1MXtt9/Ak/xpwj+F2vQwMj0/Ui8rQi1jRJgJQg6wCjMSZBQnEUYizSrOKE128OBRLF++Fnt2HzSfvSYfO0EQCis2/l3iOnWr48orL+PhSxJeZFZRptMkxEic8ZIizDJFhJkg5AD9saGQjMSZ9pxpcWYVadqbpuvq5cg01rggCEIwYBVRWlSRkdCiUHvFvMWYFmraY6aX0+sR/CPCTBBygP7Y6JDElhZlGZmuR9Cy3usRBEEINqyCSse14NICzJ/peoQIs6whwkwQcoi3sNKii0x7ybQY02ldz7qcjguCIAQrVlFGofaCUai9Zr4Ema6nl9NxwT8izAQhh1jFFUGCSxvlWb1kFFKeLrOaIAhCYUALK21adGkhpkWZLtOml7WGgn9sGzf/Jz2DIOQQLaysIZnDQUJMiTUSZOrPbqfhS6u3TF2cqK7lJgC9DkEQhGDBKqSUHKMMFTOe3WiILhJmSqCpvxAWZTTMaYgzbzFmXZfgH/GYCUIu0B8fa0imBZh3aDVd34p3WhAEoaDwFlJWgWU1Emi+QjJd3xoKGSPCTBByibfIotDbSJj5KhMEQShMaMGljbAKMe8y71DIHBFmghAA9MfI+nGiuDbCPYzpWV8uWIIgBDvWa5U1NIYzjbg2jbWekHVEmAlCgLB+lLzj2nRaX6is9QRBEIIZ63XLKrq0afzFhawhwkwQAoj14+Tro5VZuSAIQjCTmegSUZZ7RJgJQh7g/bGSj5kgCEUVbwEmgix3iDAThDwko4+XfPQEQShsZCS6RJAFAuD/xctGBlsDRLcAAAAASUVORK5CYII=)
```
Comment (jinyuwangms) [inline documentation/ServiceNowSetup.md:15]: I cannot seem to see this image, what is it? and how can customer see it?

File: documentation/ServiceNowSetup.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
+
+
+
+## How to configure Embed widget in ServiceNow
+
+
+#### Configure Openframe CTI
+1. Login to Snow instance.
```
Comment (bharath-cherukula) [inline documentation/ServiceNowSetup.md:8]: ServiceNow

File: documentation/ServiceNowSetup.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+2. Click on All. Search for openframe. Select OpenFrame > Configurations.
+3. Click on New to add a new configuration.
+4. Set the configuration as follows and click on Submit.
```
Comment (bharath-cherukula) [inline documentation/ServiceNowSetup.md:11]: Set the configuration **as shown in the screenshot below** and click on Submit.

File: documentation/ServiceNowSetup.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
...Q3AJS0NWPiXI6hjx3J/MQoPXEGhIjBbas7k4nJWyHa8/i1e3NiGoTOGIi8vCxPFGjw391Dn/pbeJN7gsJiLGlmaepJekd3znXhyWQtMV2exsEMxI83Mmm57sMPiLj+6PyfQ6HQaZF+QgIiIUNnHPRSGwtI5fYUe9PF035/TlQtkH0/S2MEYQR8ViE/Ez8fSC4pDcNP0MOjZVtr0i3BJl2HUR9pNg3HXhGQMGZ6L+24wMEWpxfo9wIGNtaiEAdN+MwITBycjd1I2rh0gn8QwjR6BN14ehWns3OzBObh2OPM82szkFzCYIhBupFAahEvNmVAY6k9g5VYRpusG4w9T0zFk8CA8cD+zMEpqsP44hVUZPUpvOb5cxSyawQPwhwcGsbDpmPbwAAxhYrw+v63r/DjHOf0NsqYKf6hGU5NicbuHwlBYXyywHkioyvpzUsbh2Q+YwH3wC1wnaUoIht4zC58ywXvznqGSAJ0dJvCkJZ/j070mPWtOuEEZvGD3s6WFbmooLshweJ1Fey22vLftx2YCjaR1S0uH1MfRsu6IFF5yz9QyPyvqTjuC9Do17ZLFhqxIpPUovVaWX+znaDl+qYT9ZakkwpWnHaMyZ+PNOX0XsqaSU1iT7lQDGhrciw8dozAU1hcLrGdnuu3P4fN5FNJywiVhKSiocHjItBQ0SU2EpJxYt4Ldbiax0SEmDtBLd8aMsnL6daUBa/60G69uBsbcPxh5C8dgttSU6AZZ8MiCeF9qbihuHGaPdATpVWw1+GQJCYsOk29K72F6dTBR4GED8EansGOQN9ndt7a8OadvExZmwMCB8aiqbMTp02dPGiQ/OkZhKKwveCQ8jj6ezv03nV3X/T+B7ONRLcMHYnoWqzVXHsIzS4/iQFEVDqzeicf/yqrXyCjMmpooB3RQtqkQ649WoWDdTry02gpkxOPaQUDupTRvoh0rlxzAgaMVzLo5ghUljnNIkCqr2I/JgOxBoWg/WIgtVFU74RCuJqz/+Dh2H2fXjs7C9Ks0aNlYiEVba1FV14LCrcfx4oLjKApCn8ZZ2JjFUd+EuvpaFOz6HgtpjlIRkP0L1uQcxBLfo/Sm4+dTWUE4fArvfHEKZXVMsHcV452XD2O73Bl9Vn54cE5/JCREh8ysODTUm1FV1Qi7YJccbZMfHaMwvgH8P7L9216urB1EAAAAAElFTkSuQmCC)
```
Comment (bharath-cherukula) [inline documentation/ServiceNowSetup.md:18]: Image not showing up.

**Comment by jinyuwangms** [comment]
Let's merge this PR as a first version and resolve comment via another PR later so that we can already start to share this to customer and refer to it.

**Review by bharath-cherukula — APPROVED** [review]

**Review by vertikapandey — APPROVED** [review]

**Pull request merged**